packages/s/surreal.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "surreal",
+  "filename": "surreal.js",
+  "description": "A lightweight JavaScript library for creating and manipulating HTML in a fluent, jQuery-like way.",
+  "keywords": [
+    "javascript",
+    "dom",
+    "manipulation",
+    "library"
+  ],
+  "homepage": "https://github.com/gnat/surreal",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/gnat/surreal.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/answerdotai/surreal.git",
```

Comment:
I'm not sure I'm a fan of using a fork of the source repository as the auto-update target here, as there is no guarantee the source code isn't being tampered with (I even see a commit in the repo that does modify the source). Please work with the original author to have it tagged or published to NPM instead.